src/plugins/intel_gpu/src/plugin/ops/convert.cpp

```diff
@@ -41,7 +41,9 @@ static void CreateConvertOp(Program& p, const std::shared_ptr<ngraph::op::v0::Co
                                       cldnn::format::any,
                                       outDataType,
                                       std::vector<float>(),
-                                      cldnn::reorder_mean_mode::subtract);
+                                      cldnn::reorder_mean_mode::subtract,
+                                      cldnn::padding(),
+                                      true);
```

Comment:
Should this be always truncate?Can we get any condition from ngraph when to make it saturate or truncate?

Reply:
Aligned offline. All "Cast" operations are to be truncated.